ansible/roles/ssl_setup/tasks/main.yml

```diff
@@ -0,0 +1,116 @@
+---
+  # Assumes that if console cert exists so does apiserver cert
+  - name: Check if SSL certs have already been generated for this instance
+    stat:
+      path: /etc/letsencrypt/archive/{{ openshift_hostname }}
+    register: certsRemote
+
+  - name: Check if SSL certs have already been generated for this instance locally
+    stat:
+      path: certs/{{ openshift_hostname }}-console-cert.pem
+    register: certsLocal
+    delegate_to: localhost
```

Comment:
If you add a line to this,  "sudo: no", then it will not error if you're not using passwordless sudo and still appears to work fine.